AubreyV/asm_system
# admin can add a new user

## Description / Purpose (checkpoints)
This PR implements the Admin Add User functionality

## Type of change
 - New feature (non-breaking change which adds functionality)

## How To Test This PR
 1. Go to Admin dashboard
 2. Click the Add User Icon
 3. Fill up the form
 4. Submit and a new User should be added to the list.
 5. The User should be able to receive an email of the activation link

 **Commands**
  - php artisan migrate:fresh

 **Routes/URL**
  - none

## Note:
 - copy the below code to your `.env`

```
MAIL_DRIVER=smtp
MAIL_HOST=smtp.gmail.com
MAIL_PORT=587
MAIL_USERNAME=micmic.saturre@gmail.com
MAIL_PASSWORD=kelkel17
MAIL_ENCRYPTION=tls
MAIL_FROM_NAME=FramgiaAdmin

```
